### ISSUE TITLE
How do I update data on react?

### ISSUE THREAD

**Issue by sudoraiku** [handsontable/handsontable#6240]
### Description
I want to upload CSV, parse it and load the parsed data on table. Currently I am importing CSV, parsing it with papaparse and storing the result on state in a parent component. And passing the csvData state as a prop to the Table component. This csvData is passed as data on HandsOnTable component. When I upload a CSV file, it always adds an empty row at 2. Everything else is working fine. I don't know, if this is the proper way to do this. How can I do it in a better way/ fix this problem?

### Steps to reproduce

1. Parse a CSV and store it in a state.
2. Pass the CSV State as a prop to another child component
3. set the CSV prop as data prop to HandsOnTable react component.

### Demo

https://codesandbox.io/s/nice-knuth-nu5do
Sample CSV file - https://gist.github.com/sudoraiku/58259d57d6fd6352af1b256997ee7396

### Your environment
* Handsontable version: 7.1.1
* Browser Name and version: Firefox
* Operating System: Ubuntu

**Comment by AMBudnik** [comment]
Hey @sudoraiku 

this issue is connected with https://github.com/handsontable/handsontable/issues/3937

Once you get rid of the `minSpareRows` it works well.

**Comment by sudoraiku** [comment]
@AMBudnik Thanks for the reference! I was able to fix it by setting `filter: false` :dancer:

**Comment by AMBudnik** [comment]
Hi @sudoraiku 

We just fixed an issue related to yours. Please update to 8.0.0-beta1 and check if it solved for you as well.

Below are the resources

```html
</style><script src="https://cdn.jsdelivr.net/gh/handsontable/handsontable@release/8.0.0-beta.1/dist/handsontable.full.js"></script><link type="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/gh/handsontable/handsontable@release/8.0.0-beta.1/dist/handsontable.full.css">
```

And this is how you can download the beta from NPM

`npm i handsontable@beta`

**Comment by AMBudnik** [comment]
The referenced issue has just been closed as solved in v 8.0.0 https://github.com/handsontable/handsontable/issues/3937
